qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -12,12 +12,20 @@
     host_script = serial_host_send_receive.py
     guest_script = VirtIoChannel_guest_send_receive.py
     Windows:
+        driver_name = vioser
         guest_script_folder = C:\
         clean_cmd = del /f /q
         tmp_dir = %TEMP%
         python_bin = python2.7
     variants:
         - @default:
+            variants:
+                - max_ports:
+                    type = virtio_serial_file_transfer_max_ports
+                    start_vm = no
+                    serials = "serial0 vs0"
+                    serial_type_vs0 = "virtserialport"
+                    virtio_serial_ports = 30
```

Comment:
The structure is better to be like the following.
- @default:
- max_ports:
......
- long_tiem_test:
......

Reply:
And do we need to test three protocals?

Reply:
> The structure is better to be like the following.
> 
> * @default:
> * max_ports:
>   ......
> * long_tiem_test:
>   ......
Yes, so I update just now, thanks.

Reply:
> And do we need to test three protocals?
I think unix_socket is enough. will add `only unix_socket` in cfg file, thanks